### PR TITLE
Test modifier for onboarding

### PR DESCRIPTION
**Pull request by itsjoekent**
#### What's this PR do?

This is a variation / modifier for onboarding to make it full screen.
We will apply the modifier via Optimizely for now on campaigns with no competition/voter reg/etc modal flow
#### How should this be reviewed?

Add -onboarding class to the onboarding div
#### Any background context you want to provide?

This _was_ 90% mobile ready but we decided to not worry about that for now, we'll only target desktop in optimizely
#### Relevant tickets

Fixes 1 part of #6929 
#### Checklist
- [ ] Documentation added for new features/changed endpoints.
- [ ] Tested on staging.
- [ ] Pinged a PM if this is a larger PR that would benefit from some additional testing love.

### CODE REVIEW

**Comment by weerd** [comment]
> Add -onboarding class to the onboarding div

Do you mean `-fullscreen`?

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss [hunk 1]
```diff
@@ -8,6 +8,45 @@
     position: relative;
   }
 
+  &.-fullscreen {
+    position: fixed;
+    z-index: 1000;
+    top: 0;
+    left: 0;
+    width: 100vw;
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss]: Did `100%` not work here?

**Comment by weerd** [comment]
Add a screenshot of what it looks like in the issue description!

**Comment by itsjoekent** [comment]
> Do you mean -fullscreen?

Uhhhhhhh :shifty: 👀

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss (hunk 1 above)
Comment (itsjoekent) [reply lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss]: I think this is leftover from mobile, but works the same for this purpose. do you prefer just 100%?

**Comment by itsjoekent** [comment]
screen shot 2016-08-25 at 12 50 32 pm

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss (hunk 1 above)
Comment (weerd) [reply lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss]: Mostly curious if there was a specific reason. Did you test in IE and other browsers to make sure it works right? I think support for `vw` units is better nowadays, but best to confirm it works right across multiple browsers!

Comment (itsjoekent) [reply lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss]: ooof good call http://caniuse.com/#search=vw

_can you imagine a world without IE..._

ill flip it to 100%

**Comment by itsjoekent** [comment]
@weerd okie dokes flipped it to percents

File: lib/themes/dosomething/paraneue_dosomething/js/components/Onboarding.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ class Onboarding extends React.Component {
     }
 
     return (
-      <div id="onboarding" className="onboarding">
```
Comment (itsjoekent) [inline lib/themes/dosomething/paraneue_dosomething/js/components/Onboarding.js:34]: dammit i commited the wrong file

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss (hunk 1 above)
Comment (weerd) [reply lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss]: Yep yep. I checked the caniuse.com info, and seems it's only partial support for `vmax` so I think we'd be in the clear with `vw` but if we're still targeting IE8 then defs not...

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss [hunk 3]
```diff
@@ -8,6 +8,45 @@
...
+      max-width: none;
+      padding-top: 64px;
+    }
+
+    .slideshow {
+      position: relative;
+      top: 50%;
+      transform: translateY(-50%);
+
+      .slide {
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss]: we use `slideshow__slide` for the specific slide and it already should have a `display: block` declaration...

Comment (itsjoekent) [reply lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss]: just checked, yep! ill just get rid of this entire selector then

**Comment by weerd** [comment]
¡OLÉ! :sheep: it!

**Comment by itsjoekent** [comment]
🚂 🚋 🚋🚋 🚋🚋 🚋🚋 🚋🚋 🚋🚋 🚋🚋 🚋🚋 🚋🚋 🚋🚋 🚋

**Pull request merged**